thunder_performance_measurement.routing.yml

```diff
@@ -0,0 +1,9 @@
+
+thunder_performance_measurement.site_info_controller_siteInfo:
+  path: '/thunder-performance-measurement/site-info'
+  defaults:
+    _controller: '\Drupal\thunder_performance_measurement\Controller\SiteInfoController::siteInfo'
+    _title: 'Get content type bundle information by some rule'
+  methods: [GET]
+  requirements:
+    _access: 'TRUE'
```

Comment:
We ship this with Thunder, so it could be enabled by site owners and would expose data without any kind of permission.
This is problematic, we should only allow admin to use this route.